### ISSUE TITLE
Hard dependency on pytest

### ISSUE THREAD

**Issue by wtbarnes** [PlasmaPy/PlasmaPy#738]
Similar to #669, but seems to have cropped up in a different place. A simple `import plasmapy` gives

```python traceback
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-2-1b30db356091> in <module>
----> 1 import plasmapy

~/anaconda3/envs/spectral-signatures-tne/lib/python3.7/site-packages/plasmapy/__init__.py in <module>
     42 from .version import githash as __githash__
     43 
---> 44 from . import formulary
     45 from . import atomic
     46 # ----------------------------------------------------------------------------

~/anaconda3/envs/spectral-signatures-tne/lib/python3.7/site-packages/plasmapy/formulary/__init__.py in <module>
----> 1 from .braginskii import *
      2 from .collisions import *
      3 from .dielectric import *
      4 from .dimensionless import *
      5 from .dispersionfunction import *

~/anaconda3/envs/spectral-signatures-tne/lib/python3.7/site-packages/plasmapy/formulary/braginskii.py in <module>
    137 from astropy import units as u
    138 from astropy.constants.si import (e, m_e, k_B)
--> 139 from plasmapy import (atomic, utils)
    140 from plasmapy.atomic.atomic import _is_electron
    141 from plasmapy.formulary.parameters import (Hall_parameter, _grab_charge)

~/anaconda3/envs/spectral-signatures-tne/lib/python3.7/site-packages/plasmapy/atomic/__init__.py in <module>
      6 from .special_particles import ParticleZoo
      7 
----> 8 from .particle_class import Particle
      9 from .particle_input import particle_input
     10 

~/anaconda3/envs/spectral-signatures-tne/lib/python3.7/site-packages/plasmapy/atomic/particle_class.py in <module>
     24     MissingAtomicDataWarning,
     25     )
---> 26 from plasmapy.atomic.parsing import (
     27     _dealias_particle_aliases,
     28     _parse_and_check_atomic_input,

~/anaconda3/envs/spectral-signatures-tne/lib/python3.7/site-packages/plasmapy/atomic/parsing.py in <module>
     15 from plasmapy.atomic.special_particles import _Particles, ParticleZoo
     16 
---> 17 from plasmapy.utils import roman, pytest_helpers
     18 
     19 from plasmapy.atomic.exceptions import InvalidElementError, InvalidParticleError, AtomicWarning

~/anaconda3/envs/spectral-signatures-tne/lib/python3.7/site-packages/plasmapy/utils/pytest_helpers/__init__.py in <module>
----> 1 from plasmapy.utils.pytest_helpers.pytest_helpers import (
      2     run_test,
      3     run_test_equivalent_calls,
      4     assert_can_handle_nparray,
      5 )

~/anaconda3/envs/spectral-signatures-tne/lib/python3.7/site-packages/plasmapy/utils/pytest_helpers/pytest_helpers.py in <module>
      1 """Test helper utilities."""
      2 import functools
----> 3 import pytest
      4 import inspect
      5 import collections

ModuleNotFoundError: No module named 'pytest'
```

**Comment by namurphy** [comment]
Thanks!  That's a good catch.  Maybe what we can do is follow astropy's example and have all of the test helper functionality in `plasmapy.tests`, and then only import `plasmapy.tests` if pytest is installed.  I might get to this in #728 (at least partially).

**Comment by StanczakDominik** [comment]
For now, to alleviate this, I'm taking the brute force approach and simply adding pytest as a hard requirement while I'm releasing 0.3.0 to solve #737 .

**Comment by StanczakDominik** [comment]
Closed by #739 !
